src/era_validator.rs

```diff
@@ -154,98 +154,3 @@ impl EraValidator for PreMergeAccumulator {
         Ok(root)
     }
 }
-
```

Comment:
@pedrohba1, usually, with something like this, you'd create an issue and open a separate PR, just FYI 👍